From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@alexandair__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

Byronr commented:
@alexandair Thanks for the update. @tysonn Please merge.

alexandair commented:
@byronr @tysonn After 25 days this is still not merged. :(

sanjeev3 committed:
Update facebook-post-moderation.md

rayne-wiselman committed:
renaming articles and updating non MVC TOC items

rayne-wiselman committed:
Update .openpublishing.redirection.json

rayne-wiselman committed:
Update .openpublishing.redirection.json

neilpeterson committed:
[AKS] get kubelet logs doc

neilpeterson committed:
toc

MohitGargMSFT committed:
(ForumsTeam) Fixes MicrosoftDocs/azure-docs#5296

(ForumsTeam) Fixes MicrosoftDocs/azure-docs#5296

Commit:
Link changes.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into vnet-article-renames-vnet

rayne-wiselman committed:
Update toc.yml

rayne-wiselman committed:
removing redirected file

TerryLanfear committed:
article update

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into new-acs

rayne-wiselman committed:
Update toc.yml

rayne-wiselman committed:
Update toc.yml

TerryLanfear committed:
fixes

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into new-acs

rayne-wiselman committed:
Update toc.yml

rayne-wiselman committed:
Update toc.yml

rayne-wiselman committed:
Update toc.yml

rayne-wiselman committed:
Update toc.yml

rayne-wiselman committed:
adding missing file

Commit:
Added the HEVC Specification and updated the TOC with a new referneces section

TerryLanfear committed:
new file and images

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into new-acs

mahesh-unnikrishnan committed:
Fix typo

Commit:
fixed toc and title

TerryLanfear committed:
TOC update

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into new-acs

rwike77 committed:
Update date

rayne-wiselman committed:
Update .openpublishing.redirection.json

rayne-wiselman committed:
fixing redirect issue

rayne-wiselman committed:
Merge branch 'rayne-march7' of https://github.com/rayne-wiselman/azure-docs-pr into rayne-march7

rayne-wiselman committed:
fixing redirect issue

rayne-wiselman committed:
fixing redirect issue 2

rayne-wiselman committed:
Update toc.yml

rayne-wiselman committed:
Update hyper-v-vmm-secondary-support-matrix.md

rayne-wiselman committed:
Update vmware-physical-secondary-support-matrix.md

Commit:
Fixed merge conflict.

Commit:
Fixes.

Commit:
Fixes.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into vnet-article-renames-vnet

Commit:
Fixes.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into vnet-article-renames-vnet

apwestgarth committed:
Update TOC.yml

Removed update documents from the toc as we need to re-release the update files to mitigate customer issues.

Commit:
Image renames and alt text.

Lisaco88 committed:
Fixed typo

Lisaco88 committed:
Fixed typo

Lisaco88 committed:
Fixed formatting

Lisaco88 committed:
Fixed formatting

Lisaco88 committed:
Fixed typo

TerryLanfear committed:
fix

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into new-acs

Commit:
Added FAQ.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into traffic-analytics2

Commit:
Fixed link.

mahesh-unnikrishnan committed:
Update active-directory-ds-networking.md

neilpeterson committed:
[AKS] remove / redirect acs jenkins doc

Commit:
Fixed link.

klaaslanghout committed:
adding the piece that non removable not support 

customer incident came in on this.

Commit:
Updated Date. The blank sections are required per the ammendment format

neilpeterson committed:
[AKS] oms update to include kubernetes secret

ktoliver committed:
Merge pull request #34887 from apwestgarth/patch-4

Update TOC.yml

Commit:
Fixed H1.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into traffic-analytics2

wesmc7777 committed:
Merge pull request #4 from MicrosoftDocs/master

sync with upstream master

wesmc7777 committed:
minor path formatting

sanjeev3 committed:
Update quick-start.md

sanjeev3 committed:
Update overview.md

PRMerger16 committed:
Merge pull request #34902 from wesmc7777/java_web_app_app_svc_linux

Java web app app svc linux

Lisaco88 committed:
Merge pull request #34834 from jimdial/vnet-article-renames-vnet

Vnet article renames.

rayne-wiselman committed:
fixing broken links

sanjeev3 committed:
Update text-moderation-api.md

sanjeev3 committed:
Update try-text-api.md

TerryLanfear committed:
fix

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into new-acs

dlepow committed:
Batch: add links to service updates

PRMerger20 committed:
Merge pull request #34899 from klaaslanghout/patch-16

adding the piece that non removable not support

ktoliver committed:
Merge pull request #34897 from neilpeterson/jenkins-redirect

[AKS] remove / redirect acs jenkins doc

rayne-wiselman committed:
Update hyper-v-azure-failover-failback-tutorial.md

daveba committed:
updating breacrumb in toc yml file to point to managed service identity

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into update-breadcrumb-file-toc

rayne-wiselman committed:
Update site-recovery-overview.md

neilpeterson committed:
updated typo

ktoliver committed:
Merge pull request #34853 from johndeu/master

Added the HEVC Specification and updated the TOC with a new refernece…

Commit:
Merge pull request #34896 from mahesh-unnikrishnan/patch-17

Update active-directory-ds-networking.md

Commit:
Merge pull request #34854 from mahesh-unnikrishnan/patch-18

Fix typo

Commit:
Merge pull request #34856 from rwike77/patch-35

Update date

mahesh-unnikrishnan committed:
Update active-directory-ds-faqs.md

Commit:
Merge pull request #34907 from dlepow/fix2

Batch: add links to service updates

PRMerger13 committed:
Merge pull request #34910 from mahesh-unnikrishnan/patch-17

Add B2B related FAQ

PRMerger14 committed:
Merge pull request #34824 from MohitGargMSFT/patch-7

(ForumsTeam) Fixes MicrosoftDocs/azure-docs#5296

ktoliver committed:
Merge pull request #34837 from TerryLanfear/new-acs

New acs

ktoliver committed:
Merge pull request #34908 from daveba/update-breadcrumb-file-toc

[Azure AD MSI] Update breadcrumb file toc

rmca14 committed:
Merge pull request #34888 from jimdial/traffic-analytics2

Image renames and alt text.

rmca14 committed:
Merge pull request #34818 from neilpeterson/kublet-logs

[AKS] get kubelet logs doc

mahesh-unnikrishnan committed:
Update active-directory-ds-faqs.md

Commit:
Removed dedicated PIP AZ articles.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into pip-az

v-geberr committed:
new concept - prediction score

Commit:
Merge pull request #34911 from mahesh-unnikrishnan/patch-17

FAQ on password lifetime policy.

v-geberr committed:
edits

mimckitt committed:
Fixes Script | Location Parameter not needed

The script provided has a Location Parameter. This is not needed as you are already deploying the Vnet into a Resource Group that is associated to a location. In addition, the GitHub repro we link to that has the examples already has these changes in place. So simply adding parity here.

douglaslMS committed:
Clarified file storage requirements per email thread.

PRMerger16 committed:
Merge pull request #34915 from MicahMcKittrick-MSFT/patch-5

(ForumsTeam) Fixes MicrosoftDocs/azure-docs#5462

v-geberr committed:
edit link

PRMerger17 committed:
Merge pull request #34916 from douglaslMS/adf-spark-service

Clarified file storage requirements per email thread.

PRMerger18 committed:
Merge pull request #34904 from neilpeterson/aks-oms-update

[AKS] oms update to include kubernetes secret

rmca14 committed:
Merge pull request #34805 from rayne-wiselman/rayne-march7

renaming articles and updating non MVC TOC items

v-geberr committed:
edits

danieldobalian committed:
fixing typos, text

iainfoulds committed:
[VM] Fixes MicrosoftDocs/azure-docs#5459

Adding sentence that clearly states you pay for each VM that is created in an Availability Set.

v-geberr committed:
edits

PRMerger16 committed:
Merge pull request #34924 from danieldobalian/master

fixing typos, text

PRMerger17 committed:
Merge pull request #34925 from iainfoulds/patch-15

[VM] Fixes MicrosoftDocs/azure-docs#5459

apwestgarth committed:
Update azure-stack-app-service-add-worker-roles.md

Added PowerShell example to scale out workers as this capability has been regressed in the compute UI in Azure Stack for some time and is a regular source of frustration for customers.

cynthn committed:
Update quick-create-portal.md

apwestgarth committed:
Update azure-stack-app-service-add-worker-roles.md

Improving quality of document

Commit:
Merge pull request #34694 from sanjeev3/master

Updates to tutorials and API articles.

apwestgarth committed:
Update azure-stack-app-service-add-worker-roles.md

Fixed build issues - author name and link to PowerShell docs

PRMerger10 committed:
Merge pull request #34927 from cynthn/patch-67

Update quick-create-portal.md

Ja-Dunn committed:
Merge pull request #34913 from jimdial/pip-az

Removed separate public IP AZ articles and redirected them to manage PIP article.

PRMerger14 committed:
Merge pull request #34928 from apwestgarth/patch-5

Update azure-stack-app-service-add-worker-roles.md

rjagiewich committed:
Merge pull request #34914 from v-geberr/0307-concept-score

0307 concept score

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

BryanTrach-MSFT committed:
Update azure-docs-pr.md

Removed the preview tag from the premium (v2) tier. Assigning feedback to author to add Isolated tier.

v-alje committed:
Merge pull request #34933 from MicrosoftDocs/FromPublicRepo

From public repo

PRMerger20 committed:
Merge pull request #34932 from BryanTrach-MSFT/patch-1

(forumsteam) fixes MicrosoftDocs/azure-docs#5444
